pytorch_lightning/overrides/data_parallel.py

```diff
@@ -99,7 +99,14 @@ def forward(self, *inputs, **kwargs):  # pragma: no cover
                 output = self.gather(outputs, self.output_device)
         else:
             # normal
-            output = self.module(*inputs, **kwargs)
+            # output = self.module(*inputs, **kwargs)
+            # lightning (ddp_cpu)
+            if self.module.training:
```

Comment:
why this change?

Reply:
This is copying the changes you made above for the case where `self.device_ids` is None. I think previously, we would never hit this code.